data/resources/gamedb.yaml

```diff
@@ -15896,6 +15896,8 @@ SLPS-02300:
     - DisableWidescreen
     - ForcePGXPCPUMode # Fixes jitter in character models.
     - ForceRecompilerICache # Fixes hangs in CD code.
```

Comment:
I wonder if this is still necessary. I played maybe 1/3 of the game without ICache and with 70k cycles, didn't experience any issues.